Sources/UI/Style/AvatarViewStyle.swift

```diff
@@ -10,10 +10,15 @@ import UIKit
 
 /// Avatars style.
 public struct AvatarViewStyle: Hashable {
+    /// Default avatar style
+    public static let `default` = AvatarViewStyle(radius: .channelAvatarRadius, placeholderFont: nil, verticalAlignment: .center)
```

Comment:
```suggestion
    public static let `default` = AvatarViewStyle()
```
We have default values in `.init`

Reply:
I believe it doesn't make sense to have a `default` property in this case, it's just an alias for `.init()`